test/lib/commands/unpublish.js

```diff
@@ -63,6 +63,23 @@ t.test('no args --force error reading package.json', async t => {
   )
 })
 
+t.test('with args --force error reading package.json', async t => {
+  const { npm } = await loadMockNpm(t, {
+    config: {
+      force: true,
```

Comment:
Did you go with `force` / deleting the entire package because it requires less mocks?

Reply:
That's cause it was copied from the `no args` test.

Reply:
It's still needed cause we're unpublishing a bare package name. The spec check happens before this.